### PR TITLE
Improve Treament Summary Report

### LINKED ISSUE #8060: Improving "Treatment summary"
**Is your feature request related to a problem? Please describe.**
The treatment summary generated in CARE needs to be improved.

eg treatment summary
[Patient Dashboard _ CARE.pdf](https://github.com/user-attachments/files/15902375/Patient.Dashboard._.CARE.pdf)

**Describe the solution you'd like**

- [x] Name the file as "Treatment summary-CARE", currently it is named as "Patient Dashboard _ CARE"
- [x] Remove "History of present illness : ---, Examination details and clinical conditions :Skin is pale....xyz, Physical Examination info : ---" from the summary
- [x] In diagnosis section first show Principle Diagnosis followed by confirmed diagnosis, provisional, unconfirmed and differential diagnosis. Do not show the heading as eg- "differential diagnosis" if there is no differential diagnosis added to the patient record
- [x] remove all sections with no data - in the above eg treatment summary, there is no data to show in "treatment". the section may not be added to the summary
- [x] relevant investigation may be changed to suggested investigations and show list of investigations suggested in consultation/ progress note (log update)
- [x] Add section for prescription- ONLY show active prescriptions
- [x] special instructions. move general instructions and special instructions to the bottom

### PR DESCRIPTION
**Pull request by khavinshankar**
## Proposed Changes

- Fixes #8060

### CODE REVIEW

File: src/Components/Facility/Consultations/Beds.tsx [hunk 1]
```diff
@@ -234,7 +234,7 @@ const Beds = (props: BedsProps) => {
             <div>
               <ButtonV2 variant="primary" type="submit">
                 <CareIcon icon="l-bed" className="text-xl" />
-                Move to bed
+                Update
```
Comment (bodhish) [inline src/Components/Facility/Consultations/Beds.tsx]: Shoudn't we move these strings to I18n config?

File: src/Components/Facility/TreatmentSummary.tsx [hunk 2]
```diff
@@ -337,3 +306,74 @@ const TreatmentSummary = (props: any) => {
...
+    <div className="border-b-2 border-gray-800 px-5 py-2">
+      <b>Diagnosis :</b>
+      <div className="mx-5">
+        {(
+          [
+            { key: "principal", title: "Principal Diagnosis" },
+            { key: "confirmed", title: "Confirmed Diagnosis" },
+            { key: "provisional", title: "Provisional Diagnosis" },
+            { key: "unconfirmed", title: "Unconfirmed Diagnosis" },
+            { key: "differential", title: "Differential Diagnosis" },
```
Comment (bodhish) [inline src/Components/Facility/TreatmentSummary.tsx]: I think we should internationalize the strings.

**Review by bodhish — CHANGES_REQUESTED** [review]

**Comment by samholics** [comment]
tested..however the IP number in patient dashboard is mentioned as OP number in the treatment summary
image
image

**Comment by khavinshankar** [comment]
@samholics fixed the above issue, can you please check again

cc: @nihal467

**Comment by samholics** [comment]
tested, working fine.  issue resolved. @nihal467 @khavinshankar

**Comment by nihal467** [comment]
@khavinshankar 

image

- [x] mention the principal diagnosis was unconfirmed, confirmed, etc... as a badge in the report

**Comment by nihal467** [comment]
LGTM

File: src/Components/Facility/TreatmentSummary.tsx [hunk 3]
```diff
@@ -52,290 +65,466 @@ const TreatmentSummary = (props: any) => {
...
+
+interface IDiagnosisSection {
+  consultationData?: ConsultationModel;
+}
+
+type DiagnosisType =
+  | "principal"
+  | "confirmed"
+  | "provisional"
+  | "unconfirmed"
```
Comment (rithviknishad) [inline src/Components/Facility/TreatmentSummary.tsx]: Import `ConditionVerificationStatuses` instead

File: src/Components/Facility/TreatmentSummary.tsx [hunk 4]
```diff
@@ -52,290 +65,466 @@ const TreatmentSummary = (props: any) => {
...
+
+  const diagnoses = useMemo(() => {
+    return consultationData?.diagnoses?.reduce(
+      (acc, curr) => {
+        if (curr.is_principal) {
+          acc.principal.push(curr);
+        } else if (
+          ["confirmed", "provisional", "unconfirmed", "differential"].includes(
+            curr.verification_status,
+          )
```
Comment (rithviknishad) [inline src/Components/Facility/TreatmentSummary.tsx]:
```suggestion
          ActiveConditionVerificationStatuses.includes(
            curr.verification_status,
          )
```

**Review by rithviknishad — CHANGES_REQUESTED** [review]

**Review by rithviknishad — APPROVED** [review]

**Review by bodhish — APPROVED** [review]
LGTM

**Pull request merged**